.pylintrc

```diff
@@ -46,7 +46,6 @@ enable=import-self,
        used-before-assignment,
        cell-var-from-loop,
        global-variable-undefined,
-       dangerous-default-value,
```

Comment:
Didn't have time to check the rest yet, but this doesn't seem right

Reply:
There's a problem with pymc-marketing and pymc-expental, as they don't use the same linting rules. So It's something I forgot to 'undo' after installing locally pymc-experimental from my local repo.